src/Microsoft.Health.ControlPlane.CosmosDb/Microsoft.Health.ControlPlane.CosmosDb.csproj

```diff
@@ -3,6 +3,7 @@
   <PropertyGroup>
     <TargetFramework>netcoreapp2.1</TargetFramework>
     <CodeAnalysisRuleSet>..\..\CustomAnalysisRules.ruleset</CodeAnalysisRuleSet>
+    <AutoGenerateBindingRedirects>true</AutoGenerateBindingRedirects>
```

Comment:
Was this change intentional?

Reply:
Not sure how these got added. Will remove it.